source/feature/openvswitch/openvswitch-spec.html.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Native Open vSwitch
+category: feature
+authors: phoracek danken edwardh
+wiki_category: Feature
+wiki_title: Features/Open vSwitch
+wiki_revision_count: 1
+wiki_last_updated: 2016-02-24
+feature_name: Native Open vSwitch
+feature_modules: Networking
+feature_status: Spec
+---
+
+# Native Open vSwitch Spec
+
+## Summary
+Open vSwitch (OVS) is a production quality multilayer virtual switch that is widely used in virtualization and dedicated HW environments.
+OVS is targeted to replace the native linux networking solutions (mainly the bridge).
+The integration of OVS into oVirt has began by introducing a special hook that implements the networking details using OVS.
+OVS is to be fully integrated (natively) into oVirt through VDSM, becoming the default networking solution.
+
+## Goals
+1. Replace native linux networking entities (e.g. bridge) with OVS on oVirt hosts.
+2. Preserve current networking requirements from oVirt hosts.
+3. Preserve or increase networking performance.
+4. Keep both switching options available, letting end users of ovirt-4.0 decide when they would like to upgrade their existing 3.6 clusters.
+
```

Comment:
this is THE motivation of moving to OVS (hence it should be mentioned here) but it is not something that is planned to take place as part of this feature during 4.0.